tests/unit/test_zowe_core.py

```diff
@@ -297,3 +383,47 @@ def test_secure_values_loading_exception(self, get_pass_func):
             prof_manager = ProfileManager()
             prof_manager.config_dir = self.custom_dir
             props: dict = prof_manager.load("ssh")
+
+
+class TestValidateConfigJsonClass(unittest.TestCase):
+    """Testing the validate_config_json function"""
+
+    def test_validate_config_json_parameterized(self):
```

Comment:
You only have two test cases, you handle both cases separately with an if statement and thus parametrization does not make sense. Rather have two separate test methods:
`test_validate_config_json_valid`
`test_validate_config_json_not_valid`

Reply:
Ok, I'll make two separate tests.